event_routing_backends/processors/tests/transformers_test_mixin.py

```diff
@@ -119,6 +119,7 @@ def test_event_transformer(self, event_filename, mocked_uuid4):
                 try:
                     self.compare_events(actual_transformed_event, expected_event)
                 except Exception as e:  # pragma: no cover
+                    print("Comparison failed, writing output to test_output for debugging")
```

Comment:
nit: debugging statement can be removed
```suggestion
```

Reply:
I added this because every time I'm in that code I am confused by what it's doing, and there is nothing in the rest of the test run that references that this functionality exists. It's handy so I want to make sure people know it's there! So I'd like to keep some version of it, but I'm happy to present it in a different way if there's a better one, though.

Reply:
It's fine to leave as-is if it's useful; I just have a knee-jerk reaction to print statements :)